### PR TITLE
Google sheets docs

### PR DESCRIPTION
**Pull request by martjanz**
https://github.com/mapcraftlabs/labs/issues/2964

### CODE REVIEW

File: google_sheets.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      Logger.log(response)
+    }
+  })
+}
+
+function getSimName(iteration) {
+  var date = new Date();
+
+  const month = ('0' + (date.getMonth() + 1)).slice(-2);
+  const day = ('0' + date.getDate()).slice(-2);
```
Comment (fscottfoti) [inline google_sheets.md:91]: What does the '0' do?  I mean, if the day is 20 does it make the string "020"?

**Review by fscottfoti — COMMENTED** [review]
This looks good thanks.  Just a couple of comments...

File: google_sheets.md (hunk 1 above)
Comment (martjanz) [reply google_sheets.md:91]: We always want to use double digits so this prepends a zero, then takes two digits from the right.

File: google_sheets.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        },
+        parcelFilter: row["parcel_filter"],
+        partitionZoomLevel: 0
+      }
+
+      if (row["workspace"].length > 0) {
+        payload.workspace = row["workspace"]
+      }
+
+      const response = MapCraft.makeLabsAPIRequest(url, payload, token)
```
[the remark this answers is not in the document]
Comment (martjanz) [reply google_sheets.md:81]: I think we should include the library on the repo, yes. I'll add it.

[the remark this answers is not in the document]
Comment (martjanz) [reply google_sheets.md:81]: [Here](https://github.com/mapcraftlabs/labs/tree/main/scripts/sheets-api/mapcraft-library)'s the library.

[the remark this answers is not in the document]
Comment (martjanz) [reply google_sheets.md:81]: On a new Sheet a library should be imported on Apps Script project using library ID (currently `1fXX4Ww_onXV6oGgAO7kTR8LlFKWkaYPyHtj4_uCchGDOYGr1J8cu1vSZ`).

image

On template Sheets or its copies the library is already linked to Apps Script project.

File: google_sheets.md (hunk 1 above)
Comment (fscottfoti) [reply google_sheets.md:91]: Got it - sneaky

File: google_sheets.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (fscottfoti) [reply google_sheets.md:81]: I think we should add that to the docs.  Do you know how to version the library?  Does it keep the same id or they have to update the script id?  Do you have to use an id or can you use a name instead etc?

**Pull request merged**